File: microservices/investment/src/main/java/com/illunex/invest/investment/persistence/entity/ReviewItem.java
package com.illunex.invest.investment.persistence.entity;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "review_item")
public class ReviewItem {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long idx;
    String content;

    @ManyToOne @JoinColumn(name = "category_idx")
    ReviewItemCategory reviewItemCategory;
}

File: microservices/invest/src/main/kotlin/com/illunex/invest/invest/persistence/entity/ReviewItem.kt
package com.illunex.invest.invest.persistence.entity

import javax.persistence.*

@Entity
@Table(name = "review_item")
data class ReviewItem(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var idx: Long? = null,
    var title: String? = null,
    var point: Int? = 0,
    var content: String? = null,
    var updateDate: String? = null,
    var deleted: Boolean? = false,

    @ManyToOne @JoinColumn(name = "category_idx")
    var category: ReviewItemCategory?

) {
    constructor() : this(null, null, null, null, null, null, null )
}
